### ISSUE TITLE
XAML Controls Gallery: Accessibility: [MAS4.3.1]: In HC mode, KB focus is not visible on File/Edit/Help buttons in StandardUICommand card

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#4275]
Tags
Accessibility, A11yCT, A11yMAS, A11yMediumImpact, MAS4.3.1, HighContrast, Sev3, XAML Controls Gallery (Debug), Win32, Wipro

Steps to reproduce the bug

1.Launch the XAML Controls Gallery (Debug) App
2.Go to the Menus and Toolbar-->StandardUIComamnd card and activate it.
3.Now navigate to the File/Edit/Help button using tab key.
4. Observe KB focus is not visible on these buttons, in HC mode.

Actual behavior
In HC mode, KB focus is not visible on File/Edit/Help buttons in StandardUICommand card

Expected behavior
Everything should visible properly in HC mode.

User Impact
People with Low vision are not able to identify the selected option when the combo box is focused.

Version Info
Microsoft OS: 21313.1000
App: XAML Controls Gallery (Debug)

https://user-images.githubusercontent.com/74585893/108679313-54c70400-74ba-11eb-9ba1-03a41e40d4a4.mp4
